### ISSUE TITLE
Navigation of usecase figures on the first page

### ISSUE THREAD

**Issue by SarahAlidoost** [eucp-project/storyboards#14]
Currently (after merging #8), there are some circles under the image of a use-case on the first page. The circles are used for navigation and implemented based on vueper-slide, see examples [here](https://antoniandre.github.io/vueper-slides/). 

First, I think that the usage of circles i.e. navigation is not intuitive. Second, the functionality does not work as it is shown in the [examples](https://antoniandre.github.io/vueper-slides/). When I click on one circle, it does not change from no-fill to a filled circle.

**Comment by Peter9192** [comment]
Well spotted, the black fill seems to be missing from our caroussel.

A bit of context: we changed the layout because we thought the caroussel was taking up too much space and was a bit noisy. We removed the navigation arrows that were overlaid on the image, and we moved the bullets on top of the figure, instead of under it.

I think we have to rethink the design of the index page once we have a few more use cases. If you already have some ideas, feel free to add them here.
